.dockerignore

```diff
@@ -0,0 +1,4 @@
+build/
+.git
+Jenkinsfile
```

Comment:
can't add testing-related files as they would be excluded by *all* images, also the testing one

Reply:
Unusual for tests to change without the code, so not much of a problem.